modules/core/src/main/java/org/apache/ignite/internal/util/IgniteUtils.java

```diff
@@ -551,6 +531,14 @@
     private static boolean devOnlyLogDisabled =
         IgniteSystemProperties.getBoolean(IgniteSystemProperties.IGNITE_DEV_ONLY_LOGGING_DISABLED);
 
+    /** List of OS specific system messages for disk full error. */
+    private static final String[] ERR_DISK_FULL = {
+            "No space left on device", // Linux
+            "There is not enough space on the disk", // Windows
+            "Not enough space", // Solaris
```

Comment:
What about MacOS?

Reply:
Anyone with MacOsX out there? it is Darwin, so must be unix message, but would be nice to  check, cant google it

Reply:
Checked: on macos it is 'Linux' message